server/routes/createPlan/qualifications/qualificationsController.test.ts

```diff
@@ -182,5 +191,38 @@ describe('QualificationsController', () => {
       expect(next).toHaveBeenCalledTimes(0)
       expect(res.redirect).toHaveBeenCalledWith(addressLookup.learningPlan.profile(id, 'education-and-training'))
     })
+
+    it('On success - mode = update - remove qualification - calls CIAG api and redirects to qualifications', async () => {
+      config.featureToggles.useNewInductionApiEnabled = false
+      req.body.removeQualification = 'LEVEL_3-Maths-A'
+      req.params.mode = 'update'
+      req.context.plan = {
+        qualificationsAndTraining: {
+          qualifications: [{}],
+        },
+      }
+
+      await controller.post(req, res, next)
+
+      expect(next).toHaveBeenCalledTimes(0)
```

Comment:
Is this assertion missing? `expect(mockCiagService.updateCiagPlan).toBeCalledTimes(1)`

Reply:
Let me double check - these tests tripped me up because the implementation is different to most of the other controllers in this respect (more logic paths that are not properly tested 😢 )

Reply:
Good spot; yes, it was missing 👍 

There is some subtlety with these tests/controller implementation, in that the CIAG Plan / Induction is only updated if a qualification has been removed. Other changes (POSTs) to this page do not appear to need up update the plan ? 🤷‍♂️ 
This code path (removing a qualification) was not tested at all previously, hence these 2 test methods appear as new tests in the diff
I'm not confident the other code paths in `handleUpdate` (adding qualifications and "existing qualifications" (not sure what that means)) are tested properly, but I was not about to pick through and add tests to stuff we've not touched